doc/util/versionButton.php

```diff
@@ -106,7 +107,7 @@ function dropSetup() {
   // Choose button color
   if (chplTitle.includes("pre-release")) {
     button.classList.add("preRelease");
-  } else if (chplTitle != "1.17") {
+  } else if (chplTitle != chplVersions[1]) {
```

Comment:
In this PR, let's actually set this back to "1.16" so that users of the documentation over the next week won't be confused.  Then when I update the web docs, I'll change this to "1.17".  I'm still trying to come up with a way to manage this file better during this version number shuffle so that we don't have days with the current documentation showing up as orange.